docs/content/docs/getting-started/quick-start.md

```diff
@@ -77,22 +77,24 @@ tkn pr logs --last -f
 
 ### Step 5: validations
 
+> **_NOTE:_** The following assumes you are running a local registry proxy (i.e. `make registry-proxy`).
+It also assumes you have run the IBM tutorial example (i.e. `make example-ibm-tutorial`).
+
 #### First some convenience exports
 
 We start by defining some variables to simplify the validation commands:
 
 ```bash
-IMAGE_URL=$(tkn pr describe --last -o json | jq -r '.spec.params[] | select(.name=="imageUrl") | .value')
+IMAGE_URL=$(tkn pr describe --last -o jsonpath='{.spec.params[?(@.name=="imageUrl")].value}')
 export IMAGE_URL=localhost:8888${IMAGE_URL#"registry.kube-system.svc.cluster.local"}
-export IMAGE_TAG=$(tkn pr describe --last -o json | jq -r '.spec.params[] | select(.name=="imageTag") | .value')
+export IMAGE_TAG=$(tkn pr describe --last -o jsonpath='{.spec.params[?(@.name=="imageTag")].value}')
 export DOCKER_IMG="${IMAGE_URL}:${IMAGE_TAG}"
-export COSIGN_KEY="k8s://tekton-chains/signing-secrets"
```

Comment:
Why removing this value? Since it used several times, it makes sense to have it no?

Reply:
Changed it to match what is currently in https://github.com/thesecuresoftwarefactory/ssf/blob/main/examples/ibm-tutorial/README.md

Reply:
Can update the README.md instead then?

Reply:
I guess I'm inclined to have fewer variables to carry around in the examples users are expected to copy and paste. The other variables are holding values that are derived at runtime whereas `COSIGN_KEY` is a static value.

Reply:
oh ok... I'm not sure I agree, and I think the opposite is more correct, but that should not be a blocker.